ros_buildfarm/wrapper/apt.py

```diff
@@ -27,6 +27,7 @@ def main(argv=sys.argv[1:]):
         'Hash Sum mismatch',
         'Unable to locate package',
         'is not what the server reported',
+        'corrupted package archive'
```

Comment:
Since the error doesn't happen on `update` but only on `install` should it be appended to the variable `known_error_strings_redo_update` in the function `call_apt_update_install_clean` instead?